### ISSUE TITLE
Can you please show hungarian loss implementation of pytorch in tensorflow. 

### ISSUE THREAD

**Issue by Gruhit13** [Leonardo-Blanger/detr_tensorflow#7]
I am also ML enthusiast who came through this paper and was willing to replicate this entire behavior in tensorflow. Fortunately I even completed the project but due to lack of my knowledge I am not able to put this last piece which resides at the heart of DETR and that is implementation of its loss calculation. I tried a lot of ways but that particular this is one which I am not able to implement using tensorflow. So if you can help me with that then it would be so great of you. 

You can contact me on my email if you want to see my implementation. My email id is: gruhitml13@gmail.com

**Comment by LucasKirsten** [comment]
Hi, I have implemented the Hungarian Loss in here: https://github.com/LucasKirsten/detr_tf2/blob/master/misc/losses.py#L76

**Comment by Gruhit13** [comment]
Thank you so much sir for replying. I think my current knowledge is not
such that I can understand the code. As I am having many doubts which I
need to answer.
Thanks once again for sharing your code with me.
